koku/masu/processor/expired_data_remover.py

```diff
@@ -154,4 +157,15 @@ def remove(self, simulate=False, provider_uuid=None, line_items_only=False):
                 removed_data = self._cleaner.purge_expired_line_item(expired_date=expiration_date, simulate=simulate)
             else:
                 removed_data = self._cleaner.purge_expired_report_data(expired_date=expiration_date, simulate=simulate)
+                with ReportManifestDBAccessor() as manifest_accessor:
+                    if not simulate:
+                        manifest_accessor.delete_cost_usage_reports_older_than(self._provider, expiration_date)
+                    else:
+                        LOG.info(
+                            """Simulated removing CostUsageReportManifest for
+                            provider type: %s before billing period: %s""",
+                            self._provider,
+                            expiration_date,
+                        )
```

Comment:
```suggestion
                    if not simulate:
                        manifest_accessor.delete_cost_usage_reports_older_than(self._provider, expiration_date)
                    LOG.info(
                        """Removing CostUsageReportManifest for
                        provider type: %s before billing period: %s""",
                        self._provider,
                        expiration_date)
```

Reply:
So, this is my first time using a multiple line suggestion. I would still make sure the spacing is correct cause python really won't be happy if its off 😂 But essentially we should log the query info regardless if its simulate or not.

Reply:
@myersCody added

Reply:
@myersCody this multi-line suggestion is very nice.